### ISSUE TITLE
Failing to install composer

### ISSUE THREAD

**Issue by arthurlogilab** [YunoHost-Apps/bookstack_ynh#4]
```
2020-01-29 08:46:41,838: DEBUG - + echo 'Install BookStack composer dependancies'
2020-01-29 08:46:41,839: DEBUG - + pushd /var/www/bookstack
2020-01-29 08:46:41,839: DEBUG - /var/www/bookstack /var/cache/yunohost/from_file/bookstack_ynh-master/scripts
2020-01-29 08:46:41,839: DEBUG - + php -r 'readfile('\''https://getcomposer.org/installer'\'');'
2020-01-29 08:46:41,839: DEBUG - + php
2020-01-29 08:46:42,141: DEBUG - All settings correct for using Composer
2020-01-29 08:46:42,141: DEBUG - The HOME or COMPOSER_HOME environment variable must be set for composer to run correctly
2020-01-29 08:46:42,141: DEBUG - + ynh_exit_properly
2020-01-29 08:46:42,141: DEBUG - + local exit_code=1
2020-01-29 08:46:42,141: DEBUG - + '[' 1 -eq 0 ']'
2020-01-29 08:46:42,142: DEBUG - + trap '' EXIT
2020-01-29 08:46:42,142: DEBUG - + set +eu
2020-01-29 08:46:42,142: DEBUG - + sleep 0.5
2020-01-29 08:46:42,644: DEBUG - + ynh_print_err '--message=!!\n  bookstack'\''s script has encountered an error. Its execution was cancelled.\n!!'
2020-01-29 08:46:42,645: WARNING - [91m[1m[ERR][0m !!
2020-01-29 08:46:42,645: DEBUG - + local legacy_args=m
2020-01-29 08:46:42,646: WARNING -   bookstack's script has encountered an error. Its execution was cancelled.
```

I think the HOME and COMPOSER_HOME missing might be the problem. Will look into other ynh packages which use composer to see if that problem is fixed somehow.

**Comment by arthurlogilab** [comment]
roundcube seems to use composer : https://github.com/YunoHost-Apps/roundcube_ynh/blob/9284bef59b50ed9ea21033a2a681525bde5dd11b/scripts/_common.sh#L45 

Is this a good solution for the above problem in your opinion ?

**Comment by ericgaspar** [comment]
The repackage of the app, now on testing #8, should fix Composer failing install 
`sudo yunohost app install https://github.com/YunoHost-Apps/bookstack_ynh/tree/testing`

**Comment by ericgaspar** [comment]
Fixed with #8
